zarf.schema.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
This is generated by the CLI now right? Do we want to check in generated schemas? My concern is that this could be very easy to miss in a PR if someone is updating the schema but didn't know/forgot to run the config-schema command.

Reply:
Is the schema being used by something discreetly? I couldn't see any changes that try to read/verify against the schema.